lke_clusters_control_plane.go

```diff
@@ -0,0 +1,94 @@
+package linodego
+
+import "context"
+
+// LKEClusterControlPlane fields contained within the `control_plane` attribute of an LKE cluster.
+type LKEClusterControlPlane struct {
+	HighAvailability bool `json:"high_availability"`
+}
+
+// LKEClusterControlPlaneACLAddresses describes the
+// allowed IP ranges for an LKE cluster's control plane.
+type LKEClusterControlPlaneACLAddresses struct {
+	IPv4 []string `json:"ipv4"`
+	IPv6 []string `json:"ipv6"`
+}
+
+// LKEClusterControlPlaneACL describes the ACL configuration
+// for an LKE cluster's control plane.
+type LKEClusterControlPlaneACL struct {
+	Enabled   bool                                `json:"enabled"`
+	Addresses *LKEClusterControlPlaneACLAddresses `json:"addresses"`
+}
+
+// LKEClusterControlPlaneACLResponse represents the response structure
+// for the Client.GetLKEClusterControlPlaneACL(...) method.
+type LKEClusterControlPlaneACLResponse struct {
+	ACL LKEClusterControlPlaneACL `json:"acl"`
+}
+
+// LKEClusterControlPlaneACLAddressesOptions are the options used to
+// specify the allowed IP ranges for an LKE cluster's control plane.
+type LKEClusterControlPlaneACLAddressesOptions struct {
+	IPv4 *[]string `json:"ipv4,omitempty"`
+	IPv6 *[]string `json:"ipv6,omitempty"`
```

Comment:
A pointer to a slice _might_ be consider an anti-pattern but in this case we need omitempty to only apply to nil values.